src/index.css

```diff
@@ -1,5 +1,4 @@
 @import '~antd/dist/antd.css';
-@import '~@audius/stems/dist/stems.css';
```

Comment:
When linking stems into the client, this import was causing React to complain that files were being imported from outside of `src`. Moving the import into `app.tsx` resolved this issue. Because stems is all css modules, this move to earlier in the list of imports shouldn't have any negative effects